src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Formatting/FormattingHelpers.cs

```diff
@@ -461,7 +461,14 @@ public static bool ParenOrBracketContainsNothing(this SyntaxToken token1, Syntax
 
         public static bool IsLastTokenInLabelStatement(this SyntaxToken token)
         {
+
+/* Unmerged change from project 'Microsoft.CodeAnalysis.CSharp.Workspaces (netstandard2.0)'
+Before:
             if (token.Kind() != SyntaxKind.SemicolonToken && token.Kind() != SyntaxKind.CloseBraceToken)
+After:
+            if (token.Kind() is SyntaxKind.SemicolonToken and not SyntaxKind.CloseBraceToken)
+*/
```

Comment:
another oddity.  not srue why these files would have any issues.

Reply:
Filed https://github.com/dotnet/roslyn/issues/56523 over this as it seems to be a semantics change.